srcpkgs/iptables/files/iptables/run

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
-[ ! -e /etc/iptables/iptables.rules ] && exit 0
-iptables-restore -w 3 /etc/iptables/iptables.rules || exit 1
+for rule in /etc/iptables/iptables.rules /etc/iptables.d/*.rules ; do
+    [ ! -e "$rule" ] && continue
```

Comment:
`-f`

Reply:
Broken symlinks fail both `-e` and `-f` checks and the previous had `-e` which is why I left it. I'm not married to `-e` (my preference is for `-f`) so updating.